epynet/network.py

```diff
@@ -381,11 +381,13 @@ def parse_vertices(self):
                 elif b'[' in line:
                     vertices = False
 
-                if b"\t" not in line or b";" in line:
+                if b";" in line:
                     continue
 
                 if vertices:
-                    components = [c.strip() for c in line.decode('utf-8').split("\t")]
+                    components = [c.strip() for c in line.decode('utf-8').split()]
```

Comment:
Do we need to use the project encoding here as well or is this always in UTF-8?

Reply:
Good point. Done.